tests/Integration/DatabaseMonitoringTest.php

```diff
@@ -212,6 +214,39 @@ public function testInjectionPeerService()
         ]);
     }
 
+    public function testOrdering()
+    {
+        try {
+            $hook = \DDTrace\install_hook(self::class . "::instrumented", function (HookData $hook) {
+                $hook->span()->service = "testdb";
+                $hook->span()->name = "instrumented";
+                $hook->span()->meta["peer.service"] = 'dbinstance';
+                $hook->span()->meta["db.instance"] = 'myDatabase';
+                $hook->span()->meta["out.host"] = 'mysql';
+                DatabaseIntegrationHelper::injectDatabaseIntegrationData($hook, 'mysql', 1);
+            });
+            self::putEnv("DD_TRACE_DEBUG_PRNG_SEED=42");
+            self::putEnv("DD_DBM_PROPAGATION_MODE=full");
+            self::putEnv("DD_SERVICE_MAPPING=phpunit:mapped-service");
+            self::putEnv("DD_ENV=prod");
+            self::putEnv("DD_VERSION=v6");
+            $traces = $this->isolateTracer(function () use (&$commentedQuery) {
+                \DDTrace\start_trace_span();
+                $commentedQuery = $this->instrumented(0, "SELECT 1");
+                \DDTrace\close_span();
+            });
+        } finally {
+            \DDTrace\remove_hook($hook);
+        }
+
+        $dbmComment = preg_replace('/^.*\/\*(.*)\*\/.*$/', '$1', $commentedQuery);
+        $dbmCommentParts = explode(',', $dbmComment);
+        $keys = array_map(function ($part) {
+            return explode('=', $part)[0];
+        }, $dbmCommentParts);
+        $this->assertSame(['dddb', 'dddbs', 'dde', 'ddh', 'ddprs', 'ddps', 'ddpv', 'traceparent'], $keys);
```

Comment:
```suggestion
        $sortedKeys = $keys;
        sort($sortedKeys);
        $this->assertSame($sortedKeys, $keys);
```
Actually testing the order, not the present keys. Otherwise anyone extending this will just add his key without thinking about the fact that it needs to be ordered.